data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -46,13 +46,13 @@ then
 		"${release}" \
 		render \
 			--output-dir=/assets/cvo-bootstrap \
-			--release-image="${release}"
+			--release-image="${release}" &&
```

Comment:
We're running under `set -e`, this shouldn't be necesary right?

Reply:
That's the point. `-e` kill the script before broken directories are removed. We stop and restart in uncertain conditions and just skip the step.

Reply:
OK right.  How about something like this instead?

```
diff --git a/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template b/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template
index 1487ed82c..b2486845b 100755
--- a/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template
+++ b/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template
@@ -36,10 +36,11 @@ CLUSTER_BOOTSTRAP_IMAGE=$(podman run --rm ${release} image cluster-bootstrap)
 
 mkdir --parents ./{bootstrap-manifests,manifests}
 
-if [ ! -d cvo-bootstrap ]
+if [ ! -f cvo-bootstrap.stamp ]
 then
 	echo "Rendering Cluster Version Operator Manifests..."
 
+	rm cvo-bootstrap -rf
 	# shellcheck disable=SC2154
 	podman run \
 		--volume "$PWD:/assets:z" \
@@ -52,6 +53,7 @@ then
 	cp cvo-bootstrap/manifests/* manifests/
 	## FIXME: CVO should use `/etc/kubernetes/bootstrap-secrets/kubeconfig` instead
 	cp auth/kubeconfig /etc/kubernetes/kubeconfig
+	touch cvo-bootstrap.stamp
 fi
 
 if [ ! -d config-bootstrap ]
```

Reply:
fine with me. Do you prefer that? What about @abhinavdahiya @wking ?

Reply:
cvo-bootstrap.finished maybe

Reply:
I'm fine with both `&8` chaining and the `.stamp` approach.